airbyte-integrations/connectors/source-amazon-ads/integration_tests/spec.json

```diff
@@ -69,6 +69,17 @@
         "items": {
           "type": "integer"
         }
+      },
+      "state_filter": {
+        "title": "State Filter",
+        "description": "Reflects the state of the Display Campaign stream as enabled, paused, or archived. If you do not populate this field, it will be ignored completely.",
```

Comment:
in description message I see "Display Campaign"
but I see that all 3 Campaigns streams affected: `display campaigns`, `product campaigns`, `brands campaigns`

Reply:
resolved